docs/book/src/intro/installation.md

```diff
@@ -65,23 +65,23 @@ anvil-zksync
 
 These URLs always fetch the **latest** published build:
```

Comment:
Should we create a `stable` tag that always points the latest release?

Reply:
Created a task to track: https://github.com/matter-labs/anvil-zksync/issues/693